easybuild/tools/filetools.py

```diff
@@ -792,6 +792,57 @@ def find_easyconfigs(path, ignore_dirs=None):
     return files
 
 
+def locate_files(files, paths, ignore_subdirs=None):
+    """
+    Determine full path for list of files, in given list of paths (directories).
+    """
```

Comment:
Since the code seem to be strictly targeted at easyconfig files, the name and description should say so.

Reply:
Nothing in this function is specific to easyconfigs. I've renamed the `ec_file` local variable to `filepath`, maybe that made you reach that conclusion?